### PR TITLE
Sieve exercise

### PR DESCRIPTION
**Pull request by glennj**
This is one of my favourites.

### CODE REVIEW

**Comment by joshgoebel** [comment]
Doesn't an optimal solution here use the square root of a limit or am I thinking of something else?￼

Or is it more important profs are clean than optimal I guess I've always thought a good mix of both. ￼
￼

**Comment by glennj** [comment]
Yeah, there are some opportunities for improvement. My feeling the proof is just that: that the problem is solvable. Unless the test suite is taking tens of seconds to complete.

File: exercises/practice/sieve/sieve.spec.wren [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import "wren-testie/testie" for Testie, Expect
+import "./sieve" for Sieve
+
+Testie.test("Hello World") { |do, skip|
```
Comment (joshgoebel) [inline exercises/practice/sieve/sieve.spec.wren:4]:
```suggestion
Testie.test("Sieve of Eratosthenes") { |do, skip|
```

**Review by joshgoebel — APPROVED** [review]

**Comment by joshgoebel** [comment]
@glennj Thanks so much, what next? :-)

**Pull request merged**